Disable buttons in forms on submit

glebm opened the pull request:
This prevents accidental double-submits.

Uses the jquery-ujs `data-disable-with`.

jayroh commented:
Man, this is way overdue. I have just one comment but other than that, this is ready to go.

glebm commented:
@jayroh What is your one comment? :smile:

jayroh commented:
It didn't come through? Jesus Christ githubs new commenting ux is
maddening.

I had mentioned that the local var in one of the views could/should be
moved to where it's assigned as a value for a data attribute.

Responding via email now so I don't have it in front of me (at the movies
about to see fantastic beasts 😄)

On Sun, Nov 20, 2016 at 11:10 AM Gleb Mazovetskiy <notifications@github.com>
wrote:

> @jayroh <https://github.com/jayroh> What is your one comment? 😄
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/thredded/thredded/pull/485#issuecomment-261787161>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABuIm_qPsK4Fzri7m_cMJz5nPC6Lo38ks5rAHD6gaJpZM4K3gS->
> .
>

glebm commented:
@jayroh I've moved the variable closer but it's a bit too long to inline it (especially since I've now replaced the `input` with a `<button>`).

Hope you've enjoyed the movie! We have high hopes for it but will be waiting for a Blu-Ray! :smile:

jayroh commented:
@glebm Fun movie! It was sorely needed considering what the world feels like these past few weeks. If you need something in the next month or so to distract you - go see it. And *Arrival* is pretty great too 😉

glebm commented:
@jayroh I know what you mean about the past few weeks. Thanks!
